workload/master_config.yaml

```diff
@@ -156,7 +156,7 @@ s3api: # basic_io
       end: 12h
     delete_percentage_per_bucket: 10
     put_percentage_per_bucket: 10
-
+  type_5_bucket_ops_negative
```

Comment:
Please add required parameters here instead keeping it empty else it will fail to parse this file.

Reply:
Done